test/test_nequip.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import numpy as np
+import openmm as mm
+import openmm.app as app
+import openmm.unit as unit
+import pytest
+
+from openmmml import MLPotential
+
+nequip = pytest.importorskip("nequip", reason="nequip is not installed")
+nnpops = pytest.importorskip("NNPOps", reason="nnpops is not installed")
+rtol = 1e-5
+platform_ints = range(mm.Platform.getNumPlatforms())
+# Get the path to the test data
+test_data_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "data")
+
+
+@pytest.mark.parametrize("platform_int", list(platform_ints))
+class TestNequIP:
+    _LENGTH_SCALE = 0.1
+    _ENERGY_SCALE = 4.184
+    
+    def testCreatePureMLSystem(self, platform_int):
+        pdb = app.PDBFile(os.path.join(test_data_dir, "toluene", "toluene.pdb"))
+        potential = MLPotential("nequip", modelPath=os.path.join(test_data_dir, "toluene", "toluene-nequip.pth"), lengthScale=self._LENGTH_SCALE, energyScale=self._ENERGY_SCALE)
+        cubicBox = np.eye(3) * 2.0 * unit.nanometers
+        pdb.topology.setPeriodicBoxVectors(cubicBox)
+        system = potential.createSystem(pdb.topology)
+        platform = mm.Platform.getPlatform(platform_int)
+        context = mm.Context(system, mm.VerletIntegrator(0.001), platform)
+        positionsOriginal = pdb.getPositions(asNumpy=True)
+        energyRef = -710491.18527 # in kJ/mol, calculated using the NequIPCalculator
+        for i in range(10):
+            positions = positionsOriginal + i * 2.0 * unit.nanometers # translate molecule to test PBC
```

Comment:
This might not catch all errors, because you only translate by whole box widths.  How about `i*0.5*unit.nanometers`?